### ISSUE TITLE
Add Modules Hash Function

### ISSUE THREAD

**Issue by tylerkrupicka** [intuit/postcss-themed#25]
This package supports [modules](https://github.com/intuit/postcss-themed#modules), basically a way to create unique names for the CSS variables created by this plugin. This makes sure that if you have two versions of a package using theme variables, they do not override each other. While we have an example of how to set this up listed, it would be interesting to add a configuration option that turns on modules without creating your own function. 

Basically, a default modules function that you can use instead of supplying your own.

This function would need to be tested and produce different hashes between different versions of the same file.

**Comment by EnzoZafra** [comment]
picking this one up
